content/en/blog/2021/discovery-selectors/index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Use DiscoverySelectors To Configure Namespaces For Your Istio Service Mesh"
```

Comment:
```suggestion
title: "Use DiscoverySelectors to configure namespaces for your Istio service mesh"
```

I prefer title case, though I'd also suggest we drop "for your Istio service mesh" as this is on the Istio blog.

Reply:
Plus, the actual field is "discoverySelectors", so I'm not sure about the capital D throughout?